mods/d2k/mod.yaml

```diff
@@ -128,8 +130,14 @@ ContentInstaller:
+	CopyFilesFromCD:
+		Movies: movies/A_BR01_E.VQA, movies/A_BR02_E.VQA, movies/A_BR03_E.VQA, movies/A_BR04_E.VQA, movies/A_BR05_E.VQA, movies/A_BR06_E.VQA, movies/A_BR07_E.VQA, movies/A_BR08_E.VQA, movies/A_BR09_E.VQA, movies/A_FINL_E.VQA, movies/A_LOSE_E.VQA, movies/A_MNTG_E.VQA, movies/H_BR01_E.VQA, movies/H_BR02_E.VQA, movies/H_BR03_E.VQA, movies/H_BR04_E.VQA, movies/H_BR05_E.VQA, movies/H_BR06_E.VQA, movies/H_BR07_E.VQA, movies/H_BR08_E.VQA, movies/H_BR09_E.VQA, movies/H_FINL_E.VQA, movies/H_LOSE_E.VQA, movies/H_MNTG_E.VQA, movies/O_BR01_E.VQA, movies/O_BR02_E.VQA, movies/O_BR03_E.VQA, movies/O_BR04_E.VQA, movies/O_BR05_E.VQA, movies/O_BR06_E.VQA, movies/O_BR07_E.VQA, movies/O_BR08_E.VQA, movies/O_BR09_E.VQA, movies/O_FINL_E.VQA, movies/O_LOSE_E.VQA, movies/O_MNTG_E.VQA, movies/G_INT1_E.VQA, movies/G_INT2_E.VQA, movies/G_MAPS_E.VQA, movies/G_PLN2_E.VQA, movies/G_PLNT_E.VQA, movies/T_TITL_E.VQA
```

Comment:
This does not work on case-sensitive file systems:

![image](https://cloud.githubusercontent.com/assets/756669/8024376/50b94662-0d31-11e5-81e2-4cdaa2fa4dca.png)

![image](https://cloud.githubusercontent.com/assets/756669/8024382/7aa68656-0d31-11e5-9b52-0d6d5201410e.png)